tests/day_4.py

```diff
@@ -0,0 +1,17 @@
+from aoc import day_4
+import pytest
+
+test_input = """2-4,6-8
+2-3,4-5
+5-7,7-9
+2-8,3-7
+6-6,4-6
+2-6,4-8"""
+
+assignment_lines = test_input.splitlines()
```

Comment:
Same as `day_3`, you could throw the `.splitlines()` on the string literal and rename it to `assignment_lines`.